src/legacy/status_im/ui/screens/ens/views.cljs

```diff
@@ -292,9 +291,7 @@
       :title    (:name account)
       :subtitle (utils/get-shortened-checksum-address (:address account))
       :chevron  true
-      :on-press #(re-frame/dispatch [:bottom-sheet/show-sheet-old
-                                     {:content (fn [] [sheets/accounts-list :from
-                                                       ::ens/change-address])}])}]))
+      :on-press #(js/alert "funcitonality no longer supported")}]))
```

Comment:
related to ens name service - 

Do I need to take caution with this still being intact? 🤔 
perhaps it's better to remove account switcher completely? maybe ens support can be taken out of old ui?